packages/spacecat-shared-rum-api-client/src/functions/form-vitals.js

```diff
@@ -133,8 +134,31 @@ function containsFormVitals(row) {
   return METRICS.some((metric) => Object.keys(row[metric]).length > 0);
 }
 
+function getParentPageVitals(dataChunks, formVitals, iframeParentMap) {
+  const iframeVitals = {};
```

Comment:
how is formVitals getting used in this func?

Reply:
I think its not necessary, I will remove it